edgeless_node/src/base_runtime/alias_mapping.rs

```diff
@@ -0,0 +1,22 @@
+/// Struct representing the updatable callbacks/aliases of a function instance.
+/// Shared between a function instance's host and guest.
+#[derive(Clone)]
+pub struct AliasMapping {
```

Comment:
Note, just to make sure we are aligned: this structure will have to be evolved so that the mapping goes from:

- String -> InstanceId

to

- String (output name) -> vector of (InstanceId, weight)

to support multiple mappings for the same output (i.e., horizontal scalability within a domain).